lib/tutorials/en_US/cookies.md

```diff
@@ -1,59 +1,98 @@
-## Cookies
+# Cookies
 
 _This tutorial is compatible with hapi v17_
 
-When writing a web application, cookies are often used to keep state about a user between requests. With hapi, cookies are flexible, secure, and simple to use.
+- [Overview](#overview)
+- [Configuring the Server](#server)
+    - [server.state()](#server.state)
+    - [route.options.state()](#options.state)
+- [Setting a Cookie](#setting)
+    - [h.state()](#h.state)
+- [Overriding Options](#override)
+- [Getting a Cookie Value](#value)
+- [Clearing a Cookie](#clearing)
 
-## Configuring the server
 
-hapi has several configurable options when dealing with cookies. The defaults are probably good for most cases, but can be changed when needed.
+## <a name="overview" /> Overview
 
-To use a cookie, first you should configure it by calling [`server.state(name, [options])`](/api#-serverstatename-options) where `name` is the name of the cookie, and `options` is an object used to configure the cookie.
+When writing a web application, cookies are often used to keep state about a user between requests. With hapi, cookies are flexible, secure, and simple to use. To use cookies in hapi, you first configure the cookie by calling `server.state()`, then set the cookie via the response toolkit.  
```

Comment:
I’d remove the sentence starting with `To use cookies in hapi …`. The next paragraph has the same information and it seems duplicated here. I’d just remove it from the overview